tests/garage/tf/policies/test_qf_derived_policy.py

```diff
@@ -0,0 +1,28 @@
+"""
+This script creates a unittest that tests Gaussian policies in
+garage.tf.policies.
+"""
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.policies import DiscreteQfDerivedPolicy
+from garage.tf.q_functions import DiscreteMLPQFunction
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscreteEnv
+
+
+class TestQfDerivedPolicy(TfGraphTestCase):
+    def test_discrete_qf_derived_policy(self):
+        env = TfEnv(DummyDiscreteEnv())
+        qf = DiscreteMLPQFunction(env.spec)
```

Comment:
can you use a simpler QFunction to test this as more of a unit test? why not just `tf.identity`?